tests/ablateLibrary/chemistry/adapt_test_targets.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
```

Comment:
How is this python being used with ablate.  Should it instead live in the chemTabRepo?

Reply:
It belongs here, it automatically adapts the tests for the model to a new model.
It dynamically writes a c++ header file.

Reply:
So it is used to generate the test files.  Lets move this out of this repo and keep it with chemTab where we assume python is available.